src/shared/utils/debounce.ts

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Why not use the npm package?

Reply:
I think I'll have to because it's making typescript fail storybook compilation :O